fitbenchmarking/fitting/plotting/plot_helper.py

```diff
@@ -8,10 +8,10 @@
 import os
 import numpy as np
 import matplotlib
-matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 import copy
 from fitbenchmarking.utils.logging_setup import logger
+matplotlib.use('Agg')
```

Comment:
This change seems to be breaking.
(This)[https://matplotlib.org/faq/usage_faq.html#what-is-a-backend] suggests that it should be reverted